components/service/fxa/src/main/java/mozilla/components/service/fxa/RustObject.kt

```diff
@@ -29,7 +29,7 @@ abstract class RustObject<T> : Closeable {
         return p
     }
 
-    protected abstract fun destroyPointer(p: T)
+    abstract fun destroyPointer(p: T)
```

Comment:
This shouldn't be protected. It's currently called from a finalizer, which needs to be fixed as well: #396